### DISCUSSION TITLE
How to get ResponseBody json from API using Workflows?

### DISCUSSION THREAD

**Discussion by natthawatsuwan** [OrchardCMS/OrchardCore#9417] [Q&A]
my workflow
![jsonworkflow](https://user-images.githubusercontent.com/15166514/117791733-d128e200-b274-11eb-8ca8-c09814050c0b.png)

**Comment by jptissot** [comment]
What are you trying to do ? Are you creating content using this workflow ?

**Reply by natthawatsuwan** [reply]
I am creating content and send data to an api at the same time.
And,I want to use json respones for CRUD content.
Thanks.

**Reply by jptissot** [reply]
Are you creating content in the same site with this workflow ? If yes, you should try to use the CreateContentTask

**Reply by natthawatsuwan** [reply]
if I want to use Liquid for get value from json.
Would there be any possibility...?
![Image 4](https://user-images.githubusercontent.com/15166514/118433999-7543da80-b706-11eb-8607-505ec30b5888.png)
